Player/Console/PlayerCommand.php

```diff
@@ -114,6 +118,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if ($input->getOption('disable-internal-network')) {
             $player->addExtension(new DisableInternalNetworkExtension());
         }
+        if ($junit) {
+            $junitExtension = new JUnitExtension();
```

Comment:
As the XML extension is currently not a requirement of the player, we should add a runtime check and display a nice warning/error message if the extension is missing.